pallets/staking/src/lib.rs

```diff
@@ -1536,8 +1689,7 @@ decl_module! {
 
             let stash_balance = T::Currency::free_balance(&stash);
             let value = value.min(stash_balance);
-            let did = Context::current_identity::<T::Identity>().unwrap_or_default();
-            Self::deposit_event(RawEvent::Bonded(did, stash.clone(), value));
```

Comment:
AFAIR this is Polymesh related change as we do want to emit and IdentityId here at the time of bonding the funds